src/pages/user_profile/UserProfile.jsx

```diff
@@ -179,7 +194,22 @@ export default function UserProfile(props) {
       defaultValue={profile.email}
       disabled={true}
     />
-    <div style={{ marginTop: '60px' }} />
+    <div style={{ marginTop: '10px' }} />
+    <p
+      style={{
+        color: '#000',
+        fontSize: '16px',
+        fontWeight: '400',
+      }}
+    >
+      Send me email notifications
+    </p>
+    <FormField
+        type={FormFieldTypes.YESNORADIOGROUP}
```

Comment:
If the choices are Yes/No, does this need to use a radio button? In a case like this, I think a checkbox is equally clear and takes up less screen space